spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +112,28 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
+
+  describe "#update" do
+    it "updates an employee in BambooHR" do
+      xml = YAML.load_file("spec/fixtures/update_employee_xml.yml")
+      response = File.new("spec/fixtures/update_employee_response.json")
+      details = JSON.parse(File.read("spec/fixtures/update_employee_details.json"))
+      url = "https://x:x@api.bamboohr.com/api/gateway.php/x/v1/employees/1234"
+
+      stub_request(:post, url).with(xml).to_return(response)
+      employee = @client.employee.update(bamboo_id: "1234",
+                                         employee_details: details)
+      expected_headers = {
+                           "content-type" => ["application/json; charset=utf-8"],
```

Comment:
@kylefdoherty don't worry about this. I'll be refactoring the specs soon enough :hourglass: